src/cli-body.ts

```diff
@@ -1,13 +1,19 @@
-let { isAbsolute, relative, join, dirname } = require('path')
-let cosmiconfig = require('cosmiconfig')
-let readPkg = require('read-pkg-up')
-let globby = require('globby')
-let yargs = require('yargs')
-let bytes = require('bytes')
-let chalk = require('chalk')
-
-let getReporter = require('./reporters')
-let getSize = require('.')
+import {
+  SizeLimitConfigOptions,
+  SizeLimitNodeOptions,
+  OutputFileEntry,
+  InputFileEntry
+} from './interfaces'
+import { isAbsolute, relative, join, dirname } from 'path'
+import cosmiconfig from 'cosmiconfig'
+import readPkg from 'read-pkg-up'
+import globby from 'globby'
+import yargs from 'yargs'
+import bytes from 'bytes'
+import chalk from 'chalk'
```

Comment:
Note, that is invalid module syntax. ES modules always must have file extensions in the end.

(This note is just to spread correct ES modules syntax)

Reply:
```js
import globby from 'globby';
console.log(typeof globby)
```

```
node --experimental-modules test-module.mjs
function
```

it works! Why is this not valid?